### DISCUSSION TITLE
Adding notes to goals

### DISCUSSION THREAD

**Discussion by bogplau** [plausible/analytics#2394] [Anything Else]
It could be useful, if it was possible to add a note to the setup goals, which should be displayed above each goal in the list.

E.g. I have now extended the number of parameters in my setup like this.

`plausible('pageview', { u: prepareUrl(["approved", "payment", "orderid", "amount", "currency"]) })`

So something like this, would be useful. Without the note, it is difficult to understand the meaning:

![goalnotes](https://user-images.githubusercontent.com/115988568/198258486-d9d07704-9737-4501-83c0-fb1c7687933a.png)

( requested via email )

**Comment by Spone** [comment]
Or maybe add a name to the goal? see https://github.com/plausible/analytics/discussions/838
